social-network-ui/src/services/api/chatApi.js

```diff
@@ -0,0 +1,93 @@
+import axios from "axios";
+import {API_URL} from "../../util/url";
+
+
+export const ChatApi = {
+    sendMessage: async (message) => {
+        try {
+            await axios.post(`${API_URL}/messages/create`, message);
+        } catch (error) {
+
+        }
+    },
+
+    getUserChats: async () => {
+        try {
+            const response = await axios.get(`${API_URL}/messages`);
+
+            return response.data;
+        } catch (error) {
+
+
+            return [];
+        }
+    },
+
+    createChat: async (userId) => {
+        try {
+            const response = await axios.get(`${API_URL}/messages/create/${userId}`);
+
+            return response.data;
+        } catch (error) {
+
+
+            return null;
```

Comment:
It would be better to have a general error handler with an error page or error modal.